### ISSUE TITLE
[Bug]: The prediction module did not load when trying to predict from a SVM model

### ISSUE THREAD

**Issue by vladimirsim** [jasp-stats/jasp-issues#3049]
### JASP Version

0.19.1

### Commit ID

84d54b934fa27731bb9eec44a4aa5f7ab0744dfd

### JASP Module

Machine Learning

### What analysis are you seeing the problem on?

Prediction

### What OS are you seeing the problem on?

Windows 11

### Bug Description

I am using the nightly build version 0.19.2.0
I trained a SVM model and tried to get predictions based on it. But the prediction table did not load. I got the following error:

This analysis terminated unexpectedly.

Error in eval(predvars, data, env): object 'Good' not found

Stack trace
jaspStateR$new(object = object, dependencies = dependencies)

initialize(...)

.mlPredictionGetPredictions(model, dataset)

.mlPredictionGetPredictions.kknn(model, dataset)

kknn:::predict.train.kknn(model[['predictive']], dataset)

kknn(formula(terms(object)), object$data, newdata, k = object$best.parameters$k, kernel = object$best.parameters$kernel, distance = object$distance)

model.frame(formula, data = train)

model.frame.default(formula, data = train)

eval(predvars, data, env)

eval(predvars, data, env)

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports

### Expected Behaviour

The prediction table should load

### Steps to Reproduce

1. Train a SVM model on a training set and save it
2. Load a new, different set, called a test set. 
3. Load the trained model and try to generate predictions
4. The prediction table will not load
Here is a printscreen:
![image](https://github.com/user-attachments/assets/b3086be2-1173-43c6-a49e-90f57198c7ed)
The trained SVM model:
[SVMTrainedModelVladimirsim.zip](https://github.com/user-attachments/files/17947010/SVMTrainedModelVladimirsim.zip)

...

### Log (if any)

[JASP 2024-11-28 10_52_36 Desktop vladimirsim.zip](https://github.com/user-attachments/files/17947017/JASP.2024-11-28.10_52_36.Desktop.vladimirsim.zip)

### More Debug Information

-------- Application Info --------
JASP Version: JASP 0.19.2
Build Branch: HEAD
Build Date: Nov 26 2024 18:09:03 (Netherlands)
Last Commit: 84d54b934fa27731bb9eec44a4aa5f7ab0744dfd

-------- Basic Info --------
Operating System: Windows 11 Version 23H2
Product Version: 11
Kernel Type: winnt
Kernel Version: 10.0.22631
Architecture: x86_64
Install Path: D:/Program Files/JASP
Platfotm Name: windows
System Local: bg_BG

-------- Extra Info --------
Current code page
Active code page: 437
Active code page: 65001

Host Name:                 SHOSHOCI
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22631 N/A Build 22631
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          359898893538
Registered Organization:   
Product ID:                00330-52813-47920-AAOEM
Original Install Date:     31.1.2023 г., 12:22:04
System Boot Time:          27.11.2024 г., 9:44:36
System Manufacturer:       LENOVO
System Model:              82LM
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: AMD64 Family 23 Model 104 Stepping 1 AuthenticAMD ~2100 Mhz
BIOS Version:              LENOVO G5CN64WW(V2.10), 6.10.2022 г.
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+02:00) Helsinki, Kyiv, Riga, Sofia, Tallinn, Vilnius
Total Physical Memory:     15 706 MB
Available Physical Memory: 8 675 MB
Virtual Memory: Max Size:  16 730 MB
Virtual Memory: Available: 6 858 MB
Virtual Memory: In Use:    9 872 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\SHOSHOCI
Hotfix(s):                 5 Hotfix(s) Installed.
                           [01]: KB5045935
                           [02]: KB5012170
                           [03]: KB5027397
                           [04]: KB5046633
                           [05]: KB5044620
Network Card(s):           2 NIC(s) Installed.
                           [01]: Realtek 8822CE Wireless LAN 802.11ac PCI-E NIC
                                 Connection Name: Wi-Fi
                                 Status:          Media disconnected
                           [02]: Realtek USB GbE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.14
                                 [02]: fe80::f73:9fc3:5374:b2df
                                 [03]: fda9:de81:d862:0:bdaa:acda:e64e:528a
                                 [04]: fda9:de81:d862:0:d3ed:28cb:8c0e:2133
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by koenderks** [comment]
Same issue as https://github.com/jasp-stats/jasp-issues/issues/3048, therefore closing this. Discussion will continue in https://github.com/jasp-stats/jasp-issues/issues/3048
